sycl/include/sycl/ext/oneapi/matrix/matrix-intel.hpp

```diff
@@ -123,7 +122,7 @@ class wi_element {
     return std::make_tuple(row, col);
 #else
     throw runtime_error("joint matrix is not supported on host device.",
-                        PI_ERROR_INVALID_DEVICE);
+                        UR_RESULT_ERROR_INVALID_DEVICE);
```

Comment:
Changes in this file are conflicting with the changes in https://github.com/intel/llvm/pull/14485
After https://github.com/intel/llvm/pull/14485 is merged, these changes should not be needed